src/main/java/org/jenkinsci/plugins/vsphere/builders/Clone.java

```diff
@@ -48,21 +50,26 @@
     private final String folder;
     private final String customizationSpec;
     private final boolean powerOn;
+    private Integer timeoutInSeconds = new Integer(TIMEOUT_DEFAULT);
```

Comment:
• Why don't you declare this as `final` here and set it in the constructor if `null`?
• Why are you using an `Integer` over a primitive `int`?